usage-rules.md

```diff
@@ -0,0 +1,534 @@
+# AshAuthentication Usage Rules
+
+## Core Concepts
+- **Strategies**: password, OAuth2, magic_link authentication methods
+- **Tokens**: JWT for stateless authentication
+- **UserIdentity**: links users to OAuth2 providers
+- **Add-ons**: confirmation, logout-everywhere functionality
+- **Actions**: auto-generated by strategies (register, sign_in, etc.)
+
+## Key Principles
+- Use `mix ash_authentication.add_strategy` for password/magic_link
+- Manual setup required for OAuth2 providers
+- Always use secrets management - never hardcode credentials
+- Enable tokens for magic_link, confirmation, OAuth2
+- UserIdentity resource required for OAuth2 strategies
+- Check existing strategies: `AshAuthentication.Info.strategies/1`
+
+## Basic Setup
```

Comment:
lets remove all mention of setup here. AshAuthentication should already be installed when these rules are present, and so things should have gone through the igniter installer. I think this will mostly confuse it.